README.md

```diff
@@ -14,7 +14,7 @@ _This list will get stale from time to time_ ;-)
 - 2010
   * SELF - Slackware Demystified (https://www.youtube.com/watch?v=Ze9oHzrrnJk)
 - 2011
-  * 2011-04 Camp KDE - Interview (https://www.youtube.com/watch?v=YIpUmPul1i4)
+  * 2011-04 **Camp KDE** - _Interview_ (https://www.youtube.com/watch?v=YIpUmPul1i4)
```

Comment:
Yeah, but best to handle the dates and their format?

Reply:
Also, what if I have a link to the video?

Reply:
yeah, lets try some things out